claims_hosp/delphi_claims_hosp/backfill.py

```diff
@@ -69,9 +72,60 @@ def store_backfill_file(claims_filepath, _end_date, backfill_dir):
         "/claims_hosp_as_of_%s.parquet"%datetime.strftime(_end_date, "%Y%m%d")
     # Store intermediate file into the backfill folder
     backfilldata.to_parquet(path, index=False)
+    return path
+
+
+def merge_existing_backfill_files(backfill_dir, backfill_file, issue_date, logger):
```

Comment:
now I think about it...it might have to be a separate script all together... the way this was implemented it would open and rewrite for every day that's missing, which is not only inefficient, it would also increase the changes of the file being corrupted with the consistent open and writing....

Note to self:
I think I want to run this not in run module but in the patch level and also modify to take in multiple days at a time.

Reply:
Yea. Honestly could be a separate issue. Also, I prettied up the diagram from earlier if that helps visualize the situation.![20240909_151759.jpg](https://github.com/user-attachments/assets/17e16c79-4163-4317-9c1d-cf712d97c7bc)

Reply:
Shower thought of changing the merge that it's done every month so the cascading change doesn't have to be an issue. It would still be an issue for the already existing ones and would consider reformatting the existing ones as a one time thing.